.github/workflows/open-datadog-agent-pr.yml

```diff
@@ -66,16 +66,6 @@ jobs:
           inv -e release.set-release-json 'nightly::OMNIBUS_SOFTWARE_VERSION' ${{ steps.new_sha.outputs.NEW_SHA }}
           inv -e release.set-release-json 'nightly-a7::OMNIBUS_SOFTWARE_VERSION' ${{ steps.new_sha.outputs.NEW_SHA }}
 
-      - name: Compute milestone
-        working-directory: datadog-agent
-        env:
-          GITHUB_TOKEN: '${{ steps.app-token.outputs.token }}'
-        id: milestone
-        run: |
-          MILESTONE_NAME=$(inv agent.version --no-include-git --no-include-pre)
```

Comment:
Github won't let me comment on an unchanged line, but since this is removed you can stop fetching the tags and do a regular shallow clone: https://github.com/DataDog/omnibus-software/pull/509/files#diff-19d144a83d8e1ad7a616ec153238e9b526b87352cace23ff094c87ea9adf5fd1L29

Reply:
Nice catch, fixed in [eaa2d9f](https://github.com/DataDog/omnibus-software/pull/509/commits/eaa2d9fc79329c56c3aaabc63fbdcb85df6e8ed8) !